internal/injector/injector_test.go

```diff
@@ -0,0 +1,128 @@
+package injector
```

Comment:
It would be interesting to add tests with actual generated struct. These would probably live in `scw` package